packages/api/src/controllers/access-control.ts

```diff
@@ -243,9 +243,15 @@ app.post(
       return res.end();
     }
 
-    const origin = req.body?.webhookPayload?.headers?.Origin;
+    let origin = req.body?.webhookPayload?.headers?.Origin;
 
     if (origin) {
```

Comment:
Shouldn't we try fetching from the Referrer if origin is false here too?

Reply:
No! Only when the origin is set as a string `null`, because that's what the browser does during redirects. If it's false then it's not coming in entirely